Makefile

```diff
@@ -165,12 +224,14 @@ test_cli: build_cli
 	cd cli && RUSTFLAGS="-D warnings" $(CARGO) test
 
 test_app_spec: RUST_VERSION=$(CORE_RUST_VERSION)
-test_app_spec: version_rustup ensure_wasm_target install_cli build_nodejs_conductor
+test_app_spec: version_rustup ensure_wasm_target install_cli build_node_conductor
```

Comment:
I don't think this is right. The native conductor is not used when running app-spec tests, but the nodejs-conductor is. This will change in the future though, but currently nodejs_conductor is a dependency for app-spec, `holochain` executable is not.